number_parser/parser.py

```diff
@@ -68,7 +70,6 @@ def number_builder(token_list):
     total_value = 0
```

Comment:
similarly, as we did with the `tokenize()` function, could you rename this `number_builder()` function to `build_number()`?